ietf-te-p2mp.tree

```diff
@@ -0,0 +1,24 @@
+module: ietf-te-p2mp
+  +--rw te!
+     +--rw tunnels
+        +--rw tunnel-p2mp* [name]
+           +--ro operational-state?    identityref
+           +--rw name                  string
+           +--rw p2mp-id?              uint32
+           +--rw identifier?           uint16
+           +--rw description?          string
+           +--rw encoding?             identityref
+           +--rw switching-type?       identityref
+           +--rw provisioning-state?   identityref
```

Comment:
check if this is operational-state?

Reply:
|     +--ro operational-state?                  identityref
     |     +--rw name                                string
     |     +--rw identifier?                         uint32
     |     +--rw description?                        string
     |     +--rw encoding?                           identityref
     |     +--rw switching-type?                     identityref
     |     +--rw admin-state?                        identityref
     |     +--rw reoptimize-timer?                   uint16